abixen-platform-core/src/main/java/com/abixen/platform/core/application/service/impl/SecuribleModelServiceImpl.java

```diff
@@ -52,13 +52,13 @@ public SecurableModel getSecuribleModel(Long securableObjectId, AclClassName acl
 
         switch (aclClassName) {
             case PAGE:
-                securibleObject = pageRepository.findOne(securableObjectId);
+                securibleObject = pageService.find(securableObjectId);
```

Comment:
final missing. Please consider return value immediately. 
it will be more readable
case Page: return
case Module: return

etc.

Reply:
This var can't be final because of the assignment in the case block.